### PR TITLE
Item Translation Fixes & Updates

### PR DESCRIPTION
**Pull request by Everade**
* Fixes various wrong item descriptions
* Added some missing translations
* Some misc. cleanup

Based on official KRO descriptions and has been confirmed on rAthena & Divine-Pride

### CODE REVIEW

File: Client Support/Renewal/2020-09-02/System/itemInfo_EN.lua [hunk 1]
```diff
@@ -17828,8 +17828,6 @@ tbl = {
 			"________________________",
 			"Random chance to inflict splash damage for 10 seconds when dealing melee physical attacks.",
 			"________________________",
-			"^663399Indestructible in battle^000000",
```
Comment (llchrisll) [inline Client Support/Renewal/2020-09-02/System/itemInfo_EN.lua:17831]: Not sure why you delete it:
`제련불가. 파괴불가.` = `Impossible to smelt. indestructible.`

File: Client Support/Renewal/2020-09-02/System/itemInfo_EN.lua [hunk 2]
```diff
@@ -43452,7 +43450,9 @@ tbl = {
 			"________________________",
 			"^0000CCType:^000000 Accessory",
 			"^0000CCWeight:^000000 10",
-			"^0000CCArmor Level:^000000 1",			"________________________",			"^0000CCRequirement:^000000",
```
Comment (llchrisll) [inline Client Support/Renewal/2020-09-02/System/itemInfo_EN.lua:43455]: Just funny how my local file is fine regarding that, but the uploaded one not xD.

File: Client Support/Renewal/2020-09-02/System/itemInfo_EN.lua [hunk 3]
```diff
@@ -88936,21 +88933,10 @@ tbl = {
...
-			"________________________",
-			"Increases ^009900Healing^000000 skills effectiveness by 5%.",
-			"________________________",
-			"^0000FFRefine Level +7^000000:",
-			"STR +2",
-			"INT +2",
-			"ATK +2%",
-			"MATK +2%",
-			"Increases ^009900Healing^000000 skills effectiveness by 1%.",
+			"MDEF +1 per 2 refine Lv.",
```
Comment (llchrisll) [inline Client Support/Renewal/2020-09-02/System/itemInfo_EN.lua:88939]: Replace it with ```
"^0000FFFor each 2 Refine Levels^000000:",
"MDEF+1",```

File: Client Support/Renewal/2020-09-02/System/itemInfo_EN.lua [hunk 4]
```diff
@@ -232699,7 +232680,8 @@ tbl = {
 			"MATK +20",
 			"Every Refine Increases MATK.",
```
Comment (llchrisll) [inline Client Support/Renewal/2020-09-02/System/itemInfo_EN.lua:232681]: If you fix something, please check other lines as well and replace this as well with:
```
"^0000FFFor each Refine Level^000000:",
"MATK + 1",
```

**Review by llchrisll — CHANGES_REQUESTED** [review]
From the looks it's fine so far except the mentioned lines.
Once you have fixed those and merged the commit I'm about to do, I will approve it :).
Thanks for the contribution.
Normally I request the other itemInfo's as well, as I update all of them at once with the same stuff.
But since it's this itemInfo, I will update the others as well.

**Review by llchrisll — CHANGES_REQUESTED** [review]
Like mentioned, here the new commit, as I updated the default project version to 2018-06-20, the folder layout also changed a bit:
0ecc52e0853ffb37c0d0d2f4edbc89c4c838bc16
Once you have updated the `Renewal/System/itemInfo_EN.lua` with your file, I will be able to merge it ^^. It might be more difficult for now, but a bit of work for ya :P.

**Comment by Everade** [comment]
Aint got no time for that.

**Comment by llchrisll** [comment]
K I will do it myself then^^.
I do it since 2019, so I'm used to long term work haha.
Also not sure what you did to fix the issue with \r since I can't view that commit correctly. I guess use \n instead but well.

**Comment by Everade** [comment]
i searched and replaced all \r(tab)  with \n\r(tab)
you can use notepad++ to do that.

**Pull request closed without merging**